index.html

```diff
@@ -57,5 +58,204 @@ <h3 class="contact-title">let's connect</h3>
 				</div>
 			</div>
 		</header>
+
+		<!-- Work section -->
+		<section class="work">
+			<div class="single-card">
+				<img
+					src="/images/SnapshootPortfolio-1.png"
+					alt="Portfolio 1 image"
+					class="p-img"
+				/>
+				<div class="about-portfolio">
+					<h1>Tonic</h1>
```

Comment:
- [ ] Kindly change the `h1` tag to any other heading tag between `h2` and `h6`. I would suggest you do this because the `h1` tag is meant to be used for the main heading of the page and it is being used for the logo already. I hope this makes sense 😀

Reply:
Done